sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseFacade.java

```diff
@@ -42,6 +42,8 @@
 
 	List<CaseDataDto> getAllActiveCasesAfter(Date date);
 
+	List<CaseDataDto> getAllActiveCasesAfter(Date date, Boolean includeAdditionalParams);
```

Comment:
Rename new parameter to "includeExtendedChangeDateFilters" and add a comment that explains what exactly these additional filters are (do the renaming for all other methods where this parameter occurs as well)

Reply:
done